src/core/core/model/story.py

```diff
@@ -39,6 +40,7 @@ class Story(BaseModel):
     comments: Mapped[str] = db.Column(db.String(), default="")
     summary: Mapped[str] = db.Column(db.Text, default="")
     news_items: Mapped[list["NewsItem"]] = relationship("NewsItem")
+    links: Mapped[list["StoryLink"]] = relationship("StoryLinks", back_populates="story", cascade="all, delete-orphan")
```

Comment:
This is overly complicated.


Switch it to:     `links: Mapped[list[str]] = db.Column(JSON)`

There you could either store a list of links, or a list of dict [{ news_item_id: link }]